README.md

```diff
@@ -1,14 +1,12 @@
-React Visibility Sensor
-====
+# React Visibility Sensor
```

Comment:
These markdown changes came from the prettier commit hook. I noticed that the commit hook wasn't covering the `.jsx` test file, so I updated it to include that